altanalyze3/utilities/parser.py

```diff
@@ -147,6 +149,35 @@ def get_parser(self):
             action="store_true"
         )
         self.add_common_arguments(intron_parser)
+
+        # Protein Domain Annotation parser
+        protein_coordinates_parser = subparsers.add_parser(
+            "proteincoordinates",
+            parents=[parent_parser],
+            help="Get Protein to Domain annotations"
+        )
+        protein_coordinates_parser.set_defaults(func=get_protein_coordinates)
```

Comment:
I don't see where this function `get_protein_coordinates` is imported

Reply:
it was not committed.